pkgs/misc/cups/drivers/fxlinuxprintutil/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, lib, fetchzip, dpkg, autoPatchelfHook, cups, tcl, tk, xorg, makeWrapper }:
+let
+  debPlatform =
+    if stdenv.hostPlatform.system == "x86_64-linux" then "amd64"
+    else if stdenv.hostPlatform.system == "i686-linux" then "i386"
+         else throw "Unsupported system: ${stdenv.hostPlatform.system}";
+in
+stdenv.mkDerivation rec {
+  name = "fxlinuxprintutil-${version}";
+  version = "1.1.1-1";
+
+  src = fetchzip {
+    url = "https://onlinesupport.fujixerox.com/driver_downloads/fxlinuxpdf112119031.zip";
+    sha256 = "1mv07ch6ysk9bknfmjqsgxb803sj6vfin29s9knaqv17jvgyh0n3";
+    curlOpts = "--user-agent Mozilla/5.0";  # HTTP 410 otherwise
+  };
+
+  patches = [ ./fxlputil.patch ./fxlputil.tcl.patch ./fxlocalechk.tcl.patch ];
```

Comment:
don't think I can appropriately review patches for `tcl`. did you author them @delan

Reply:
yeah those were my own — fxlputil.tcl.patch replaces the code that looks for Tcl packages in the working directory and /usr/lib or /usr/lib64 with /nix/store/fxlinuxprintutil/lib, while fxlocalechk.tcl.patch replaces the code that looks for X11’s locale.alias in /usr/share/X11/locale or /usr/lib/X11/locale with /nix/store/libX11/share/X11/locale